src/node/desktop/src/main/gwt-callback.ts

```diff
@@ -173,11 +200,15 @@ export class GwtCallback extends EventEmitter {
     });
 
     ipcMain.on('desktop_show_folder', (event, path: string) => {
-      GwtCallback.unimpl('desktop_show_folder');
+      void shell.openPath(path)
+        .then((value) => {
+          if (value)
+            logger().logErrorMessage(value);
+        });
```

Comment:
I'm not entirely sure what the `void` here is doing. This is what the linter automatically suggested to me, so I've left it

Reply:
If you add on a `.catch` it isn't necessary to have the `void`. The `void` is just a hint to ESLint that you're aware you're not handling errors.